### DISCUSSION TITLE
Support ticket no response

### DISCUSSION THREAD

**Discussion by CyberOdSE** [community/community#47275] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, can anyone help me out?

Dear Team,

Recently I tried to re-activate my github educational developer pack. I previously claimed it on my other account (**l), I have downgraded my account on (**) and tried to re-activate on this account, but gives the error the benefits already have been claimed before.

Please help me out,

**Comment by SilasRodrigues19** [comment]
Hey @CyberOdSE 

Support takes a while to respond to tickets, you need to be patient and wait.

**Reply by AshleyJackson** [reply]
@SilasRodrigues19 What's your experience with their response times?

I've had one open for 7 days with no response.

**Reply by ghost** [reply]
It's interesting, I had a ticket that I opened to help resolve account access when returning to a previous employer. The support team responded within 24 hours and were extremely helpful. Now my account was flagged seemingly out of the blue and I'm extremely limited as to what I can do here. Ticket has been sitting unacknowledged for 9 days now. I understand exercising patience and higher volumes of support requests, but this seems excessive. It does not seem like a short term or isolated [issue](https://github.com/community/community/discussions/23989) either

**Comment by Hercztonic** [comment]
@CyberOdSE they are not responding still for me.
